src/plugins/d3/renderer/hooks/useChartOptions/y-axis.ts

```diff
@@ -49,6 +50,16 @@ const getAxisLabelMaxWidth = (args: {axis: PreparedAxis; series: ChartKitWidgetS
     }).maxWidth;
 };
 
+function getAxisMin(axis?: ChartKitWidgetAxis, series?: ChartKitWidgetSeries[]) {
+    const min = axis?.min;
+
+    if (typeof min === 'undefined' && series?.some((s) => s.type === 'bar-x')) {
```

Comment:
Why we need this condition `series?.some((s) => s.type === 'bar-x')`?

Reply:
It is for this type of visualization that it is important that the axis starts with 0 - this is the only way to build a rectangle with a minimum value. For lines or points it is not so important.